Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1891,7 +1891,7 @@ void AXObjectCache::onScrollbarFrameRectChange(const Scrollbar& scrollbar)
         return;
 
     if (RefPtr axScrollbar = get(const_cast<Scrollbar*>(&scrollbar)))
-        m_geometryManager->cacheRect(axScrollbar->objectID(), enclosingIntRect(axScrollbar->relativeFrame()));
+        m_geometryManager->cacheRectIfNeeded(axScrollbar->objectID(), enclosingIntRect(axScrollbar->relativeFrame()));
```

Comment:
I think there might be a pattern for unused results?

If nothing else I think putting (void) at the start makes it explicit that you're ignoring the return value